staramr/tests/integration/detection/test_AMRDetection.py

```diff
@@ -25,10 +26,12 @@ def setUp(self):
         blast_databases_repositories = AMRDatabasesManager.create_default_manager().get_database_repos()
         self.resfinder_dir = blast_databases_repositories.get_repo_dir('resfinder')
         self.pointfinder_dir = blast_databases_repositories.get_repo_dir('pointfinder')
+        self.plasmidfinder_dir = blast_databases_repositories.get_repo_dir('pointfinder')
```

Comment:
I think this should be set to `plasmidfinder` and not `pointfinder`.

Reply:
Resolved in commit d36d011